torch/ao/quantization/observer.py

```diff
@@ -1092,12 +1092,12 @@ def _adjust_min_max(
         # Therefore, we avoid such division operation.
         downsample_rate = int(
             torch.ceil(
-                (combined_max - combined_min) * upsample_rate / (self.max_val - self.min_val)
+                ((combined_max - combined_min) / (self.max_val - self.min_val)) * upsample_rate
             ).item()
         )
-        e = downsample_rate * (self.max_val - self.min_val) / upsample_rate - (combined_max - combined_min)
+        e = downsample_rate / upsample_rate * (self.max_val - self.min_val) - (combined_max - combined_min)
```

Comment:
Are downsample_rate and upsample_rate ever close to each other? 

If they're equal I think this will be resolve to (1) * (something close to infinity) which I think still might overflow?

Reply:
that isn't an issue, 1*(something representable by float) won't overflow